test/unit/composite-view/composite-view.child-view-container.spec.js

```diff
@@ -21,13 +21,9 @@ describe('composite view - childViewContainer', function() {
       this.CompositeView = Marionette.CompositeView.extend({
         childView: this.View,
         template: this.templateFn,
-        ui: {foo: '#foo'},
```

Comment:
this change to the test is actually related to this bug https://github.com/marionettejs/backbone.marionette/issues/3431